astromodels/functions/functions_1D/powerlaws.py

```diff
@@ -343,6 +343,83 @@ def evaluate(self, x, K, piv, index, xc):
 
         return result * unit_
 
+      
+class CPL(Function1D, metaclass=FunctionMeta):
```

Comment:
The use of CPL could be confusing as there is already a cutoff power law function. Moreover, we typically spell out the function name to be clear for users.

Reply:
fixed: CPL -> Cutoff_powerlaw_Ep